# req
string left_right
float64[] data
---
# res
bool result
